src/commands/list.js

```diff
@@ -8,10 +8,17 @@ var Table = require("cli-table");
 var _ = require("lodash");
 var logger = require("../logger");
 
+/**
+ * This command is deprecated in favor of `projects:list` command
+ * TODO: Remove this command 6 months after launching `projects:list` command
```

Comment:
Can this todo call out that date? It's already launched, so say 6 months from today-ish? (Calling out "after launching `projects:list` is redundant, given that it's deprecated in favor of that command>0

```suggestion
 * TODO: Remove this command after 02/09/2020.
```

Reply:
Done